### DISCUSSION TITLE
Update of macOS version.

### DISCUSSION THREAD

**Discussion by h4ckee** [Eloston/ungoogled-chromium#2186] [General]
Hi guys.
When are you planning to update binary release for macOS? `92.0.4515.131-1` version is pretty outdated already, no picure-in-picture mode, most of extensions are incompatible and aren't being offered for installation on https://chrome.google.com
BR,
Sergey.

**Comment by ghost** [comment]
Same question from me, will macOS get an update? :)

**Comment by networkException** [comment]
There is a maintainer for macOS who I assume has not found the time yet to migrate development to the ungoogled-software/ungoogled-chromium-macos repository. Binaries can be found in the releases here https://github.com/kramred/ungoogled-chromium-macos or using brew

**Reply by h4ckee** [reply]
Thank you friend!
